integration/test-integration/src/test/resources/apiYaml/aws_lambda_api.yaml

```diff
@@ -0,0 +1,144 @@
+type: api
+version: v4.1.0
+data:
+  id: 58036d9f-64cb-46fc-b14e-c4db149686e5
+  name: AwsLambda
+  context: /test
+  version: 1.0.0
+  provider: admin
+  lifeCycleStatus: CREATED
+  responseCachingEnabled: false
+  cacheTimeout: 300
+  hasThumbnail: false
+  isDefaultVersion: false
+  isRevision: false
+  revisionId: 0
+  enableSchemaValidation: false
+  type: HTTP
+  transport:
+    - http
+    - https
+  tags: []
+  policies:
+    - Unlimited
+  authorizationHeader: Authorization
+  securityScheme:
+    - oauth2
+    - oauth_basic_auth_api_key_mandatory
+  visibility: PUBLIC
+  visibleRoles: []
+  visibleTenants: []
+  mediationPolicies: []
+  subscriptionAvailability: CURRENT_TENANT
+  subscriptionAvailableTenants: []
+  additionalProperties: []
+  additionalPropertiesMap: {}
+  accessControl: NONE
+  accessControlRoles: []
+  businessInformation: {}
+  corsConfiguration:
+    corsConfigurationEnabled: false
+    accessControlAllowOrigins:
+      - '*'
+    accessControlAllowCredentials: false
+    accessControlAllowHeaders:
+      - authorization
+      - Access-Control-Allow-Origin
+      - Content-Type
+      - SOAPAction
+      - apikey
+      - Internal-Key
+    accessControlAllowMethods:
+      - GET
+      - PUT
+      - POST
+      - DELETE
+      - PATCH
+      - OPTIONS
+  websubSubscriptionConfiguration:
+    enable: false
+    secret: ""
+    signingAlgorithm: SHA1
+    signatureHeader: x-hub-signature
+  createdTime: "1667536348147"
+  lastUpdatedTime: 2022-11-04 10:08:18.468
+  endpointConfig:
+    endpoint_type: awslambda
+    access_method: stored
+    amznRegion: us-east-1
+    amznAccessKey: TEST123ACCESS321KEY
+    amznSecretKey: AWS_SECRET_KEY
+  endpointImplementationType: ENDPOINT
+  scopes: []
+  operations:
+    -
+      id: ""
+      target: /*
+      verb: GET
+      authType: Application & Application User
+      throttlingPolicy: Unlimited
+      scopes: []
+      usedProductIds: []
+      operationPolicies:
+        request: []
+        response: []
+        fault: []
+    -
+      id: ""
+      target: /*
+      verb: PUT
```

Comment:
Do we need all the HTTP operation types relevant to the `/*` resource ?

Reply:
Fixed by e0f92a0